recipes/RoutingHashHalogen/dev/index.js

```diff
@@ -0,0 +1 @@
+require("../../../output/RoutingHashHalogen.Main/index.js").main();
```

Comment:
Just a "best practices" kind of thing, but shouldn't the first line in this file be `"use strict";`?

```javascript
"use strict";
require("../../../output/RoutingHashHalogen.Main/index.js").main();
```

Reply:
Possibly, but I don't think we'll run into to much trouble with this one-liner. But no strong feelings either way. But if we do change it, we should change it everywhere (including the framework templates).

Reply:
Even if it won't, I'd rather it be there in case we decide to add more to it at some point for who knows what reason. At the very least, it would add credibility to this project because we're using best practices.